src/librand/distributions/ziggurat_tables.rs

```diff
@@ -12,8 +12,8 @@
 // algorithm. Autogenerated by `ziggurat_tables.py`.
 
 pub type ZigTable = &'static [f64; 257];
-pub static ZIG_NORM_R: f64 = 3.654152885361008796;
-pub static ZIG_NORM_X: [f64; 257] =
```

Comment:
i thought taking the address does give it's own memory slot, it just gives it an anonymous name.